content/lang/pt-br/guide/forms.md

```diff
@@ -15,15 +15,15 @@ Formulários no Preact funcionam de forma muito parecida ao React, exceto por n
 A documentação do React sobre [Componentes "Controlados"](https://facebook.github.io/react/docs/forms.html#controlled-components) e [Componentes "Não-Controlados"](https://facebook.github.io/react/docs/forms.html#uncontrolled-components)
 é imensamente útil para entender como tomar formulários HTML, que tem um fluxo de dados bidirecional, e fazer uso deles a partir do contexto de um renderizador Virtual DOM baseado em Componentes, que geralmente tem fluxo de dados unidirecional.
 
-Geralmente, você deve tentar usar Componentes _Controlados_ o tempo todo. Contudo, quando criando Componentes _standalone_ ou fazendo _wrappers_ para bibliotecas externas pode ser útil utilizar seu Componente como um ponto de monagem para funcionalidades não-Preact. Nesses casos, Componentes "Não-Controlados" encaixam muito bem para tal propósito.
+Geralmente, você deve tentar usar Componentes _Controlados_ o tempo todo. Contudo, quando criando Componentes _standalone_ ou fazendo _wrappers_ para bibliotecas externas pode ser útil utilizar seu Componente como um ponto de montagem para funcionalidades não-Preact. Nesses casos, Componentes "Não-Controlados" encaixam muito bem para tal propósito.
 
 ## Checkboxes & Radio Buttons
 
-Checkboxes e radio buttons (`<input type="checkbox|radio">`) podem inicialmente causar confusão quando criando formulários controlados. Isto se deve a, num ambiente não controlado, tipicamente permitirmo o navegador "alternar" ou "checar" um _checkbox_ ou um botão _radio_ para nós, esperando pela mudança e reagindo de acordo com o novo valor.
-Contudo, tal técnica não transiciona bem para a visão de mundo onde a UI é sempre atulizada automáticamente em resposta a mudanças de estado e de _props_.
+Checkboxes e radio buttons (`<input type="checkbox|radio">`) podem inicialmente causar confusão quando criando formulários controlados. Isto se deve a, num ambiente não controlado, tipicamente permitindo o navegador "alternar" ou "checar" um _checkbox_ ou um botão _radio_ para nós, esperando pela mudança e reagindo de acordo com o novo valor.
+Contudo, tal técnica não transita bem para a visão de mundo onde a UI é sempre atualizada automáticamente em resposta a mudanças de estado e de _props_.
```

Comment:
`transicionar` is not wrong, it's a verb, but 'transitar' actually sounds better! good catch